README.md

```diff
@@ -1,21 +1,3 @@
 # extendr.github.io
 
-This repository contains a dummy R project which allows us to
-build a website for Extendr using `pkgdown`.
-
-To build the site locally:
-
-```r
-remotes::install_github("r-lib/pkgdown")
-devtools::install()
-pkgdown::build_site()
-```
-
-There is a github action that builds the site and deploys to
-the `gh-pages` branch when pushing to `main`.
-
-On linux, you will also need:
-
-```shell
-sudo apt install fontconfig libfreetype2-dev libfontconfig1-dev libharfbuzz-dev libfribidi-dev pandoc
-```
+Repository for the **extendr** website built using [Quarto](https://quarto.org/).
```

Comment:
can you add that we are supposed to push _site and also that the website can be previewed with `quarto preview`?

Reply:
right now i have _site/ in gitignore because i included a github action to render the site, so i think we just need to make sure the github pages settings are pointing to that github action branch, right?

Reply:
if we want to render locally, we'll need to change the output directory to "dir" i think - github doesn't like "_site"

Reply:
Oh. I'm just repeating what I was told to do in the user guide.